bears/apertium/ApertiumLintBear.py

```diff
@@ -36,7 +36,8 @@ class ApertiumLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         apertiumlint_config: str=''):
+                         apertiumlint_config: str = '',
```

Comment:
Hm I actually liked the old pep8 recommendation, you sure we don't want to change our coafile instead?

Reply:
this was never the old pep8 recommendation.
pep8 always requested this afaik, but it was never implemented.

Reply:
Ah really? However, this is a very impacting change, you don't wanna change our coafile?

Reply:
I dont want to have this repo and coalib both violating pycodestyle rules unless they are wrong.  Better to have one which passes, and the other catches up.

Reply:
:+1: